geometry_msgs/Point world_pos
uint64 id
